### PR TITLE
[Android] Make db size configurable

### PR DESCRIPTION
**Pull request by alexkuttig**
Summary:
---------

Since I do need a larger max db size, I added an optional parameter to gradle properties. This makes the maximum db size configurable, while keeping the standard size of 6L * 1024L * 1024L when not giving any value. (like discussed in #83 and #98 )

Test Plan:
----------

1.) Use test app
2.) write > 6MB data to async storage
3.) change android/gradle.properties of testapp and uncomment the last line
4.) make new build
5.) write additional data

### CODE REVIEW

**Comment by alexkuttig** [comment]
@Krizzu I made a new Pull Request to keep the files clean.

File: example/android/gradle.properties [hunk 1]
```diff
@@ -16,3 +16,6 @@
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
+
+# If you know what you're doing, you can uncomment this line and increase the size of db
```
Comment (Krizzu) [inline example/android/gradle.properties]: Let's be more gentle :)

```suggestion
# This is an example of how you can change default DB size (6MB) to 10MB
```

**Comment by alexkuttig** [comment]
Here we go :-)

**Comment by tido64** [comment]
LGTM. I'm not as savvy on Android as I'd like to be so if @Krizzu says to merge, then so be it. 👍

**Comment by mjmasn** [comment]
BTW this was already possible via `onCreate()` in `MainApplication.java`:
```java
@Override
public void onCreate() {
    super.onCreate();
    long size = 25L * 1024L * 1024L;
    ReactDatabaseSupplier.getInstance(getApplicationContext()).setMaximumSize(size);
}
```

Not sure if it's worth adding that to the documentation as well?

**Comment by Krizzu** [comment]
@mjmasn Hey,

I don't think it's worth adding. Setting values in `gradle.properties` are more user friendly IMO.

**Pull request merged**